lib/main.dart

```diff
@@ -185,14 +238,13 @@ class _MyHomePageState extends State<MyHomePage> {
         ],
         onTap: (index) {
           _pageController.animateToPage(index,
-              duration: const Duration(milliseconds: 300),
-              curve: Curves.easeOutCubic);
+              duration: const Duration(milliseconds: 300), curve: Curves.easeOutCubic);
```

Comment:
`dartfmt` here would make this whitespace-only change disappear.

Reply:
Yeah I don't know why it didn't do that 🤔

Reply:
addressed manually